### PR TITLE
feat(core): settings sdk added

### PR DESCRIPTION
**Pull request by Schavras**

### CODE REVIEW

**Comment by Schavras** [comment]
@JohnBelly95 Also added the utils lib from loom-v2

File: apps/api/src/core/settings/settings.service.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { SettingsEntity } from '../entities/settings.entity';
+
+@Injectable()
+export class SettingsService {
+	constructor(@InjectRepository(SettingsEntity) private settingsRepository: Repository<SettingsEntity>) {
+	}
+
+	private static checkSettingsIntegrity(settings: SettingsEntity[]): void {
+		if ( settings.length === 0 ) {
+			throw new BadRequestException('SETTINGS_NOT_INITIALIZED', 'Settings object is not initialized. Call create settings first');
```
Comment (Schavras) [inline apps/api/src/core/settings/settings.service.ts:14]:
```suggestion
			throw new BadRequestException('SETTINGS_NOT_INITIALIZED', 'Settings object is not initialized. Call updateSettings first');
```

**Review by Schavras — COMMENTED** [review]
suggestion from review

File: libs/utils/src/i18n/states.ts [hunk 2]
```diff
@@ -0,0 +1,7771 @@
+import { LoomText } from '@loom/core-shared';
```
Comment (JohnBelly95) [inline libs/utils/src/i18n/states.ts:1]:
```suggestion
import { Translatable } from '@nima/interfaces';
```

File: libs/utils/src/i18n/countries.ts [hunk 3]
```diff
@@ -0,0 +1,3260 @@
+import { LoomText } from '@loom/core-shared';
```
Comment (JohnBelly95) [inline libs/utils/src/i18n/countries.ts:1]:
```suggestion
import { Translatable } from '@nima/interfaces';
```

**Review by JohnBelly95 — CHANGES_REQUESTED** [review]
Looks good, needs minor fixes

**Review by JohnBelly95 — APPROVED** [review]
LGTM 🎆🎆🎆

Squash and merge

**Pull request merged**